index.html

```diff
@@ -1,31 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>PORTFOLIO</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet"/>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>PORTFOLIO</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
-  </head>
-  <body>
     <header>
+    <nav class="header-nav-phone">
+        <a href="#" class="header-logo">
+                Welcome
+        </a>
+        <button type="button" class="menu-btn" id="nav-toggle">
+            <img src="./images/Union.png" alt="menu link">
+        </button>
+        <div class="nav-menu" id="nav-menu">
+            <div class="btn-close-nav" id="nav-close">
+                <i class=" fa fa-times"></i>
+              </div>
+              <ul>
+                  <li class="nav-item">
+                      <a href="#works-id" class="link-portfolio">
+                          Portfolio
+                      </a>
+                  </li>
+                  <li class="nav-item">
+                      <a href="#about-myself-id" class="link-about">
+                          About
+                      </a>
+                  </li>
+                  <li class="nav-item">
+                      <a href="#footer-container-id" class="link-contact">
+                          Contact
+                      </a>
+                  </li>
+              </ul>
```

Comment:
In order to have the same drop-down menu as your Figma design I recommend you to make the following changes:
- [ ] Kindly, delete the `Cancel Icon` button
- [ ] The Welcome on the left side should not appear
- [ ] When I click on Portfolio it is giving me an error when it should take me to the projects section, kindly check the link.
 Your design:

![review1](https://user-images.githubusercontent.com/53347966/203569267-13673830-51f7-49bf-bfaf-d7527b128d69.JPG)

Figma design:

![review2](https://user-images.githubusercontent.com/53347966/203569326-4ea0d739-02d0-4028-aa53-5c6d412cea4f.JPG)